### PR TITLE
🙈 code: default shoud use umi@2 branch

### PR DESCRIPTION
**Pull request by chenshuai2144**
##### Checklist

- [x] `npm test` passes
- [x] tests are included
- [ ] documentation is changed or added
- [ ] commit message follows commit guidelines

##### Description of change

- any feature?
- close https://github.com/umijs/umi/ISSUE_URL

### CODE REVIEW

**Comment by ycjcl868** [comment]
ci 挂了

File: packages/umi-build-dev/src/plugins/commands/block/download.js [hunk 1]
```diff
@@ -98,7 +98,7 @@ const getAntdVersion = ref => {
   if (process.env.BLOCK_REPO_BRANCH) {
     return process.env.BLOCK_REPO_BRANCH;
   }
-  return ref;
+  return 'umi@2';
```
Comment (ycjcl868) [inline packages/umi-build-dev/src/plugins/commands/block/download.js]: 感觉硬编码了

Comment (chenshuai2144) [reply packages/umi-build-dev/src/plugins/commands/block/download.js]: 故意的，
process.env.BLOCK_REPO_BRANCH 是唯一的口子

**Comment by chenshuai2144** [comment]
ci 是因为 react 的描述文件导致的，没必要专门修一下。

**Comment by ycjcl868** [comment]
我来修下 ci

**Comment by ycjcl868** [comment]
merge 下 2.x 应该好了

**Pull request merged**